### DISCUSSION TITLE
🚀 Feature Request:表单草稿问题

### DISCUSSION THREAD

**Discussion by flowerwOw0316** [alovajs/alova#548] [Feature Request]
### What problem does the new feature need to solve?

可不可以增加部分字段存起来,部分字段又不进行存储呢,这样页面恢复或者刷新的时候仅部分字段需要还原,比如增加个exclude:[a,b,c]代表这3个字段不进行存储

### How important is it to you?

Better to have

**Comment by MeetinaXD** [comment]
没有很理解你的意思，能否写一段代码表达你期望的用法？

**Reply by flowerwOw0316** [reply]
好的，比如
![image](https://github.com/user-attachments/assets/bb73d0d6-48e2-4d12-9ad7-6d63b90409bf)
这样在存储的时候，就会只存b和c

**Reply by MeetinaXD** [reply]
了解你的意思了

1. `useForm` 提供了 `onRestore` 事件，用于在 form 从持久化中恢复后执行自定义逻辑，你可以在这个事件中手动实现所期望的效果
2. 暂时还没有收到其他用户提出类似的需求。而且这个需求过于具体，恐怕普适性不强

综上，我们暂时不考虑添加这个功能。有任何问题欢迎继续沟通

**Reply by flowerwOw0316** [reply]
好的大佬，那目前对于我这种特殊的情况有没有什么最佳的解决思路呢，可以利用上alova的特性的，不然我只能是通过watch监听form解构出来的属性进行特殊处理了。

**Reply by MeetinaXD** [reply]
我的方案差不多也是这样，当然可以配合`updateForm` 使用 `onRestore`，效果是类似的
